rlkit/torch/sac/policies.py

```diff
@@ -69,16 +70,22 @@ def __init__(
         # so it does not need to be attached to the graph
         # TODO: this does not seem very PyTorch-esque, neither does `eval_np`
         # investigate this more later
-        self.np_z = np.zeros(self.latent_dim)
```

Comment:
Should we maybe just initialize np_z to None?

Reply:
in the future, we probably won't be setting z to 0, when we are sampling from the prior for exploration.

Reply:
it is being set to None right now.